core/embed/rust/src/ui/component/text/op.rs

```diff
@@ -258,4 +271,6 @@ pub enum Op<T: StringType> {
     CursorOffset(Offset),
     /// Force continuing on the next page.
     NextPage,
+    /// Render the following text in a chunkified way. None will disable that.
+    Chunkify(Option<(Chunks, i16)>),
```

Comment:
Should probably have separate operation for changing the line spacing as these are really a different thing.

Reply:
Done in [7f4ea77](https://github.com/trezor/trezor-firmware/pull/3237/commits/7f4ea773963e7e86969c25065f6428c46c663929)